### PR TITLE
Change to new homepage in various other files

### PR DESCRIPTION
**Pull request by ann0see**
I also found these pages.

Could you also link the new site in this repo please?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 Jamulus - Internet Jam Session Software
 =======================================
-<img align="left" src="src/res/homepage/mediawikisidebarlogo.png"/>
+<img align="left" src="src/res/homepage/mediasidebarlogo.png"/>
```
Comment (corrados) [inline README.md:7]: The logo does not show up anymore:
grafik

Comment (ann0see) [reply README.md:7]: Sorry. You have to change the file name back to mediawikisidebarlogo.png

Comment (ann0see) [reply README.md]: Done.

**Comment by corrados** [comment]
When I click on this:
grafik
I get:
grafik

**Comment by ann0see** [comment]
Yes. Which site would you like to be linked? Getting Started?

**Comment by corrados** [comment]
Good question. It was irritating for me to get to a "Page overview" page. It looks like the link is broken. Maybe this one: https://jamulus.io/wiki/Software-Manual but I am not sure if this is what we want.

@gilgongo What is your opinion?

**Comment by gilgongo** [comment]
Seeing as we may be re-arrangng the docs a bit in future, maybe just say they are available at jamulus.io for now.

**Comment by ann0see** [comment]
Ok. Done

**Pull request merged**